Example/Tests/TransactionBytes/EthContractCallBytesTests.swift

```diff
@@ -0,0 +1,67 @@
+/**
+Copyright 2018 Timofey Solonin
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import CryptoSwift
+import Nimble
+import Quick
+@testable import Web3Swift
+
+final class EthContractCallBytesTests: XCTestCase {
+
+    func testContractCallIsEncodedCorrectly() {
+        expect{
+            try EthContractCallBytes(
```

Comment:
@biboran Seven ctor parameters seems to be too much. What do you think about splitting the tested class into smaller ones?

Reply:
@driver733 part of the requirement for the transaction signing is an ability to reproduce it by specifying every dependency. There are 7-8 mathematically separate dependencies for the transaction signing.

Do you see any way to retain that functionality and reduce the number of parameters?

Reply:
@biboran @driver733 lets accept that some functions require that number of parameters due to Ethereum architecture. We could code domain wrappers for them later after experimenting with real DApps. If it is ok then let's merge this.